#!/usr/bin/env python
PACKAGE = "ommpc_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("command_or_hover", bool_t, 0, "A Boolean parameter",  False)
gen.add("land_enabled", bool_t, 0, "A Boolean parameter",  False)
gen.add("takeoff_enabled", bool_t, 0, "A Boolean parameter",  False)

exit(gen.generate(PACKAGE, "ommpc_controller", "fsm_change")) 